phpunit.xml.dist

```diff
@@ -10,7 +10,7 @@
          failOnWarning="true"
 >
     <php>
-        <server name="SYMFONY_PHPUNIT_VERSION" value="7.5" />
+        <server name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
```

Comment:
```suggestion
```

Reply:
Removing this line entirely and letting composer decide the best version for PHPUnit to install does the trick...

Reply:
Actually we should remove the entire block right here, not just the server version. e.g.
```
L12:    <php>
L13:        <server name....... />
L14:    </php
```

Reply:
see 895364cd7a265814b496321fa6b8a5b95d008e88